web/gui-v2/src/components/ListViewTable.jsx

```diff
@@ -135,9 +135,56 @@ const AGGREGATE_SUM_COLUMNS = [
 ];
 
 
+// Determine whether a given row matches the filters and/or selected group
+const filterRow = (row, filters, selectedGroup) => {
+  const filterKeys = Object.keys(filters);
+
+  if ( selectedGroup !== NO_SELECTED_GROUP ) {
+    if ( ! groupsList[selectedGroup].members.includes(row.CSET_id) ) {
+      return false;
+    }
+  }
+
+  for ( const colDef of columnDefinitions ) {
+    if ( !filterKeys.includes(colDef.key) ) {
+      continue;
+    }
+
+    // Extract the appropriate value from the row, as defined for the column
+    // (for example, the `value` key of the `ai_pubs` column).
+    const rowVal = colDef?.extract?.(row[colDef.key]) ?? row[colDef.key];
+
+    if ( colDef.type === "dropdown" ) {
+      if ( filters?.[colDef.key].length > 0 ) {
+        if ( ! filters?.[colDef.key].includes(rowVal) ) {
+          return false;
+        }
+      }
+    } else if ( colDef.type === "slider" ) {
+      if ( filters?.[colDef.key].length !== 2 ) {
+        console.error(`Invalid filter value for range filter '${colDef.key}': expecting [min, max], got ${JSON.stringify(filters?.[colDef.key].get)}`);
+        continue;
+      }
+
+      const [min, max] = filters[colDef.key];
+      if ( rowVal < min || ( max < 100 && max < rowVal) ) {
+        return false;
+      }
+    } else if ( colDef.type === "stock" ) {
+      // TODO: Figure out how we're filtering the `market_list` column
```

Comment:
Yeah, we have `market_list` and `full_market_links` and `market_filt`. I think you already found this but this is the set of exchanges we filter to: https://github.com/georgetown-cset/parat/blob/version2/web/scripts/retrieve_data.py#L63C1-L63C15